src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -277,6 +277,61 @@ public int VideoFrameRate
             }
         }
 
+        /// <summary>
+        /// Gets or sets the encoder bitrate of the current media source.
+        /// </summary>
+        /// <remarks>
+        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.<br/>
+        /// </remarks>
+        /// <value>A value that specifies the encoder bitrate.</value>
```

Comment:
as usual, it would be better to include unit, I guess its bits per second (bit/s)

Reply:
I followed the type of Native FW.